projects/1.16/assets/enchantment-descriptions/enchdesc/lang/zh_cn.json

```
@@ -1,46 +1,84 @@
+{
+  "_comment": "模组本地化",
+  "tooltip.enchdesc.addedby": "由%s添加",
+  "tooltip.enchdesc.activate": "按住%s%s%s查看魔咒描述。",
+  "enchdesc.fatalerror": "错误：请查看日志文件",
+  "key.enchdesc.show": "显示描述",
+  "key.enchdesc.title": "魔咒描述",
+  
+  "_comment": "原版魔咒描述",
+  "enchantment.minecraft.protection.desc": "减少大多数种类的伤害。",
+  "enchantment.minecraft.fire_protection.desc": "减少火焰伤害。同时减少着火时的燃烧时间。",
+  "enchantment.minecraft.feather_falling.desc": "减少摔落伤害和来源于末影珍珠的传送伤害。",
+  "enchantment.minecraft.blast_protection.desc": "减少爆炸伤害，也会降低爆炸产生的击退效果。",
+  "enchantment.minecraft.projectile_protection.desc": "减少来源于弹射物的伤害（例如箭、火球等）。",
+  "enchantment.minecraft.respiration.desc": "延长水下呼吸时间，增强水下视觉效果。",
+  "enchantment.minecraft.aqua_affinity.desc": "加快水下挖掘速度。",
+  "enchantment.minecraft.thorns.desc": "对攻击者造成伤害。",
+  "enchantment.minecraft.sharpness.desc": "增加物品的伤害值。",
+  "enchantment.minecraft.smite.desc": "对亡灵生物造成额外伤害（例如僵尸、骷髅等）。",
+  "enchantment.minecraft.bane_of_arthropods.desc": "对节肢生物造成额外伤害（例如蜘蛛、蠹虫等）。",
+  "enchantment.minecraft.knockback.desc": "增加击退的距离。",
+  "enchantment.minecraft.fire_aspect.desc": "攻击生物时造成额外的火焰伤害。",
+  "enchantment.minecraft.looting.desc": "击杀生物时会掉落更多战利品。",
+  "enchantment.minecraft.efficiency.desc": "增加工具的挖掘速度。",
+  "enchantment.minecraft.silk_touch.desc": "被挖掘的方块会掉落它们的本体而不是应该掉落的物品。",
+  "enchantment.minecraft.unbreaking.desc": "减缓工具的耐久消耗速率。",
+  "enchantment.minecraft.fortune.desc": "部分方块会掉落额外物品（例如煤矿石、钻石矿石等）。",
+  "enchantment.minecraft.power.desc": "增加弓发射的箭矢的伤害。",
+  "enchantment.minecraft.punch.desc": "增加弓箭击退的距离。",
+  "enchantment.minecraft.flame.desc": "点燃被箭命中的目标。",
+  "enchantment.minecraft.infinity.desc": "发射普通箭矢不会消耗箭（需要物品栏内至少有一根箭）。",
+  "enchantment.minecraft.luck_of_the_sea.desc": "增加钓鱼时获得宝藏战利品的几率。",
+  "enchantment.minecraft.lure.desc": "减少钓鱼的等待时间。",
+  "enchantment.minecraft.depth_strider.desc": "增加在水下时的移动速度。",
+  "enchantment.minecraft.frost_walker.desc": "在水上行走时会产生霜冰。",
+  "enchantment.minecraft.mending.desc": "用经验来修补工具和护甲的耐久度。",
+  "enchantment.minecraft.binding_curse.desc": "阻止有此附魔的物品从护甲栏移除。",
+  "enchantment.minecraft.vanishing_curse.desc": "如果带着有此附魔的物品死亡，该物品会被销毁。",
+  "enchantment.minecraft.sweeping.desc": "增加横扫攻击的伤害。",
+  "enchantment.minecraft.loyalty.desc": "允许三叉戟被投掷后自动返回。",
+  "enchantment.minecraft.impaling.desc": "增加对水生生物的伤害。",
+  "enchantment.minecraft.riptide.desc": "允许玩家跟随三叉戟向前发射。只有在雨中或水中才有效。",
+  "enchantment.minecraft.channeling.desc": "允许三叉戟在雷暴中召唤闪电。",
+  "enchantment.minecraft.multishot.desc": "向相似方向发射额外箭矢。",
+  "enchantment.minecraft.quick_charge.desc": "增加弩的装填速度。",
+  "enchantment.minecraft.piercing.desc": "允许弹射物穿透生物。",
+  "enchantment.minecraft.soul_speed.desc": "在灵魂方块上时增加移动速度。",
+  
+  "_comment": "Travel Anchors Descriptions",
+  "enchantment.travel_anchors.teleportation.desc": "允许附魔的物品代替旅行权杖使用。",
+  "enchantment.travel_anchors.range.desc": "增加最大传送范围。",
+
+  "_comment": "ColdsEnchants",
+  "enchantment.coldenchants.summoner.desc": "有概率生成一个帮助你战斗的守卫者。",
+  "enchantment.coldenchants.cat_paw.desc": "苦力怕会因为恐惧远离装备者。",
```

```suggestion
  "enchantment.coldenchants.cat_paw.desc": "苦力怕会因为恐惧远离穿戴者。",
```
没测过这个模组，如果是只能应用于盔甲的话推荐改成这个